certbot/renewal.py

```diff
@@ -38,6 +38,9 @@ def renewal_conf_files(config):
     """Return /path/to/*.conf in the renewal conf directory"""
     return glob.glob(os.path.join(config.renewal_configs_dir, "*.conf"))
 
+def renewal_file_for_certname(config, certname):
+    """Return /path/to/certname.conf in the renewal conf directory"""
+    return os.path.join(config.renewal_configs_dir, "{0}.conf".format(certname))
```

Comment:
I don't think certbot cares about being readable (I assume you mean in the permissions sense?). I like the checking for the filename existing though, especially because it'll have nice symmetry with the renew all lineages version.